gpu_ocean/SWESimulators/gpu_kernels/ocean_noise.cu

```diff
@@ -417,15 +417,17 @@ __global__ void geostrophicBalance(
         const int eta_ty = ty+1;
 
         // Find Coriolis parameter
-        const float s = ti / (float) nx_;
-        const float t = tj / (float) ny_;
+        // Discarding ghost cells, and 0.5 offset to reach the corner of the domain
+        const float s = (ti+2.0f-0.5f) / (float) nx_;
```

Comment:
Her er det pluttselig plus to?

Reply:
Good catch! Her ble det feil - skulle være -2+0.5